crates/codegen/runtime/npm/package/wasm/generated/interfaces/nomic-foundation-slang-cst.d.ts

```diff
@@ -106,26 +294,62 @@ export class NonterminalNode {
   asTerminalNode(): undefined;
   isTerminalNode(): false;
```

Comment:
seems like these are also missing doc comments. But maybe that is more related to #1166 and can be done in a separate PR.